docs/trace/reporting-exceptions/Program.cs

```diff
@@ -30,7 +30,7 @@ public static void Main()
     {
         using var tracerProvider = Sdk.CreateTracerProviderBuilder()
             .AddSource("MyCompany.MyProduct.MyLibrary")
-            .SetSampler(new AlwaysOnSampler())
+            .ConfigureSampler(x => x.SetDefaultSampler(new AlwaysOnSampler()))
```

Comment:
I think it should be

```csharp
.ConfigureSampler(x => x.Clear().SetDefaultSampler(new AlwaysOnSampler()))
```

Can you elaborate on why you suggest using the "builder API" instead of `SetSampler` if the user would like to always use one explicit sampler?

Reply:
Fixed in fb1156464964461b5629def1e8305960044d1dc7.

New API provide broader functionalities so it should be used in all examples in the repository.